Dockerfile

```diff
@@ -8,6 +8,7 @@ COPY ./requirements.txt /tmp
 RUN apk update \
 	&& apk upgrade \
 	&& apk add --no-cache bash \
+	&& apk add build-base \
```

Comment:
why was `build-base` required? It's useful to capture the reasoning on the PR

Reply:
This is needed  to have compilers like gcc for pycryptodome. I will check on how to remove them post installation of  pycryptodome